ci/dev-env-install.sh

```diff
@@ -39,7 +39,7 @@ step "Building dev-env dependencies"
 # to make it retry itself. See https://github.com/NixOS/nix/issues/2794 for the issue requesting
 # this feature.
 for i in `seq 10`; do
-    nix-build nix -A tools -A cached 2>&1 | tee nix_log
+    nix-build nix -A tools -A cached 2>&1 | tee nix_log || true
```

Comment:
If Nix does fail, do we get a failing exit code? Or just continue always now?

Reply:
We don’t look at the exit code but we look at the error message so we don’t always retry.

Reply:
@cocreature How about something like `if !nix-build ...` or `unset failed; nix-build ... || failed=1` to make sure that we don't silently ignore `nix-build` errors which might cause confusing errors down the line? You can then still check for the log content to decide whether to retry.

Reply:
Good idea, changed.